apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -89,6 +90,9 @@
 _UNS_OUTGEST_COLUMN_NAME_1D = "values"
 _UNS_OUTGEST_COLUMN_PREFIX_2D = "values_"
 
+_TILEDBSOMA_TYPE = "soma_tiledbsoma_type"
+_DATAFRAME_ORIGINAL_INDEX_NAME = "soma_dataframe_original_index_name"
```

Comment:
maybe suffix this with `_json` to indicate that it is JSON-formatted data

Reply:
(adding `_json` to the metadata key, so it’s obvious to an otherwise-unfamiliar reader of the dataset)